src/main/cli/settings.c

```diff
@@ -294,11 +294,15 @@ static const char * const lookupTablePwmProtocol[] = {
 static const char * const lookupTableLowpassType[] = {
     "PT1",
     "BIQUAD",
+    "PT2",
+    "PT3"
```

Comment:
Generally good practice to include the trailing comma. That way if the list is later extended the diff will not include changes to the previous lines. For example you see above that the `"BIQUAD",` option doesn't chow a change since it properly already had the trailing comma.